setup/schemas.sql

```diff
@@ -0,0 +1,77 @@
+USE library_management_system;
+
+IF NOT EXISTS
+    (
+        SELECT TOP 1 1
+        FROM INFORMATION_SCHEMA.COLUMNS
+        WHERE TABLE_SCHEMA = 'dbo'
+            AND TABLE_NAME = 'book'
+            AND COLUMN_NAME = 'book_id'
```

Comment:
be consistent with your convention 
you either write all of your queries like this 
or like this select statment or the other way 

        SELECT
            TOP 1 1
        FROM
            INFORMATION_SCHEMA.COLUMNS
        WHERE
            TABLE_SCHEMA = 'dbo'
            AND TABLE_NAME = 'book'